autogpt/commands/file_operations.py

```diff
@@ -298,10 +305,10 @@ def list_files(directory: str) -> list[str]:
     "download_file",
     "Download File",
     '"url": "<url>", "filename": "<filename>"',
-    CFG.allow_downloads,
+    CFG.allow_downloads,  # TODO find a way to get rid of this
```

Comment:
using a lambda function wasn't working. It's fine, let's merge this big boy in and then immediately fast follow on this CFG.